internal/activity/database/activity_queries.go

```diff
@@ -0,0 +1,23 @@
+package database
+
+const (
+	createActivity = "create activity"
+	deleteActivity = "delete activity by uuid"
+	getActivity    = "get activity by uuid"
+	listActivity   = "list activities"
+	updateActivity = "update activity by uuid"
+)
+
+func queriesActivity() map[string]string {
+	return map[string]string{
+		createActivity: `INSERT INTO
+			activities (content_uuid, name, description, content_type, taxonomy)
+			VALUES ($1, $2, $3, $4, $5) RETURNING *`,
+		deleteActivity: `UPDATE activities SET deleted_at = NOW() WHERE uuid = $1 AND deleted_at IS NULL`,
+		getActivity:    `SELECT * FROM activities WHERE uuid = $1 AND deleted_at IS NULL`,
+		listActivity:   `SELECT * FROM activities WHERE deleted_at IS NULL`,
```

Comment:
if deleted_at will have the concept of archived, then we should be able to filter by that value

Reply:
True, I guess that I left a comment related to the soft-delete filtering in the other PR. Just in case, I'll summarize it here: We need to discuss how we are going to filter and when we are going to filter the soft-deleted records.